### ISSUE TITLE
Manual response validation breaking change in 1.0.0->1.1.0 upgrade

### ISSUE THREAD

**Issue by MrBananaLord** [ahx/openapi_first#214]
Hi!

I just upgraded to 1.0.0 and then to 1.1.0. I am using the manual response validation (more or less) as follows:
```ruby
# in my middleware
definition = OpenapiFirst.load(...)
response = @app.call(env)
definition.request(Rack::Request.new(env)).response(Rack::Response[*response]).validate!
```

My response validation breaks in 1.1.0 with `NoMethodError: undefined method 'join' for #<ActionDispatch::Response::RackBody:...` on `@rack_response.body.join` in `OpenapiFirst::RuntimeRequest` ([here](https://github.com/ahx/openapi_first/blob/7234597e1bbb98a034804de582a30d45d9e9364e/lib/openapi_first/runtime_response.rb#L55)) because my `@rack_response.body` is a `ActionDispatch::Response::RackBody`.

[In the rack repo I found that the body is usually expected to respond to each or call](https://github.com/rack/rack/blob/3897649e8740e560a5fa142f972121a119b26b5c/lib/rack/response.rb#L45) so maybe we could go back to [using each](https://github.com/ahx/openapi_first/blob/2b5b69ce8d491a8c803b04f3a2f9b3c0bf213c53/lib/openapi_first/response_validation/validator.rb#L49)? :)

**Comment by ahx** [comment]
Hi. Thanks for the feedback. I have to admit that I was afraid something could go wrong here, but Rack::Lint did not complain (…). My fault.
I am wondering if there is some tooling in Rack that makes this part simpler. …

I will try to come up with a fix shortly to release 1.1.1

**Comment by ahx** [comment]
Fix released in 1.1.1
